src/components/ProjectionsEditor.tsx

```diff
@@ -25,6 +26,7 @@ export default function ProjectionsEditor() {
         return (
           <Fragment>
             <ProjectionsChart ref={ref} {...options} height={350} />
+            <CustomizableChart ref={ref} {...options} height={350} chartType="line" />
```

Comment:
Maybe we can enable `chartType` as an option to user can choose. What you think?